packages/babel-preset-jest/index.js

```diff
@@ -5,14 +5,14 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-module.exports = () => ({
-  plugins: [
-    require.resolve('babel-plugin-jest-hoist'),
-    // TODO: replace all the below with a single preset in Jest 26: https://github.com/facebook/jest/pull/9774
-    require.resolve('@babel/plugin-syntax-object-rest-spread'),
-    require.resolve('@babel/plugin-syntax-bigint'),
-    require.resolve('@babel/plugin-syntax-class-properties'),
-    require.resolve('@babel/plugin-syntax-logical-assignment-operators'),
-    require.resolve('@babel/plugin-syntax-numeric-separator'),
-  ],
-});
+const plugins = [
+  require.resolve('babel-plugin-jest-hoist'),
+  // TODO: replace all the below with a single preset in Jest 26: https://github.com/facebook/jest/pull/9774
+  require.resolve('@babel/plugin-syntax-object-rest-spread'),
+  require.resolve('@babel/plugin-syntax-bigint'),
+  require.resolve('@babel/plugin-syntax-class-properties'),
+  require.resolve('@babel/plugin-syntax-numeric-separator'),
```

Comment:
You should also add optional chaining and nullish coalescing (Babel 7.8.0).

Reply:
Oh nevermind, it's not supported by Node.js. I should add it to the preset for the future.

Reply:
It'll be supported in node 14 though, which is due to release April 21st, so might make sense to add regardless